extensions/elation/actions/createCareGap/createCareGap.ts

```diff
@@ -0,0 +1,47 @@
+import { Category, type Action } from '@awell-health/extensions-core'
+import { makeAPIClient } from '../../client'
+import { type settings } from '../../settings'
+import { dataPoints, fields, FieldsValidationSchema } from './config'
+
+export const createCareGap: Action<
+  typeof fields,
+  typeof settings,
+  keyof typeof dataPoints
+> = {
+  key: 'createCareGap',
+  category: Category.EHR_INTEGRATIONS,
+  title: 'Create care gap',
+  description: 'Create a care gap in Elation.',
+  fields,
+  previewable: true,
+  dataPoints,
+  onActivityCreated: async (payload, onComplete, onError): Promise<void> => {    
```

Comment:
JFYI - we are deprecating `onActivityCreated` and started using `onEvent.` The latter has some helpful helper functions and passes data as an object rather than separate function params.

See below for reference implementations:
- https://github.com/awell-health/awell-extensions/blob/main/extensions/bland/actions/sendCall/sendCall.ts#L20
- https://github.com/awell-health/awell-extensions/blob/main/extensions/bland/actions/sendCall/__tests__/sendCall.test.ts#L18

Reply:
Aware of that. I decided not to refactor as part of this PR as all actions I saw in elation were using the `onActivityCreated` pattern.